apps/testingaccessibility/prisma/schema.prisma

```diff
@@ -41,9 +41,11 @@ model Coupon {
   usedCount                Int               @default(0)
   percentageDiscount       Decimal           @db.Decimal(3,2)
   restrictedToProductId    String?
+  bulkPurchaseId           String?
+  bulkPurchase             Purchase?         @relation(name: "BulkCoupon", fields: [bulkPurchaseId], references: [id], onDelete: NoAction, onUpdate: NoAction)
   merchantCoupon           MerchantCoupon?   @relation(fields: [merchantCouponId], references: [id], onDelete: Cascade)
   product                  Product?          @relation(fields: [restrictedToProductId], references: [id], onDelete: Cascade)
-  purchases                Purchase[]
+  purchases                Purchase[]        @relation("StandardPurchase")
```

Comment:
`bulkPurchase` and `purchases` have to be disambiguated with the reference so a name is required on either end in the relation